saleor/search/documents.py

```diff
@@ -67,12 +71,34 @@ class Meta:
 @orders.doc_type
 class OrderDocument(DocType):
     user = fields.StringField(analyzer=email_analyzer)
+    first_name = fields.StringField()
+    last_name = fields.StringField()
 
     def prepare_user(self, instance):
         if instance.user:
             return instance.user.email
         return instance.user_email
 
+    def prepare_first_name(self, instance):
+        user = instance.user
+        if user:
+            if user.first_name:
```

Comment:
In addition to what @Pacu2 suggested, could you add a little docstring summarizing what the method does so we keep the code clean?